src/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -86,7 +79,7 @@ const hostDiscoveryStepValidationsMap: WizardStepValidationMap = {
   },
   host: {
     allowedStatuses: ['known', 'disabled'],
-    groups: ['hardware'],
+    groups: ['hardware', 'network'],
```

Comment:
I am not sure what is the intention here, the network validations should not be checked in host-discovery step. Is there some specific validation we want to check from the networking group? If so add it to the `validationIds`.

Reply:
IIRC the `connected` validation belongs to the network group

Reply:
Right, that's why it is explicitly specified in `validationIds`. `groups` array is used to specify that we want to check all validations in these groups instead of having to name all of them in `validationIds` array. So in this case the 'network' group should not be included here.

Reply:
Removed.